views/auth/local/login.ejs

```diff
@@ -1,25 +1,55 @@
-<div class="grid-container">
-  <div class="grid-row">
+<div class="grid-container login-block">
+
+  <div class="grid-row ">
     <div class="tablet:grid-col-6 tablet:grid-offset-3 desktop:grid-col-4 desktop:grid-offset-4">
-      <h1 class="text-center">Log in</h1>
-      <% if (locals.unauthorized) { %>
-      <div class="alert alert-danger">
-        Invalid email and/or password.
+      <div class='text-center'>
+        <div>
+          <h2 class='hospital-dest-title'>Hospital</h3>
+          <h2 class='hospital-dest-title'>destination tool</h3>
+        </div>
+        
+          <p> Start by entering your login information below.</p>
       </div>
-      <% } %>
+
+     
       <form method="post" action="/auth/local/login" class="usa-form">
         <div class="usa-form-group">
-          <label for="username" class="usa-label">Email</label>
-          <input type="email" name="username" value="<%= locals.username %>" class="usa-input" />
+          <label for="username" class="usa-label" id='email-label' ><h6 style='color:red; padding:0; display: inline; padding-right:0.5%'>*</h6>Email</label>
```

Comment:
I'd typically try to avoid inline styles- especially since, in the case of this required asterisk styling, it's repeated again below in the next field.

I'm going to experiment by making the required asterisk a 100% css content style to clean this up and will push up the change...